fabric-rendering-v1/src/main/java/net/fabricmc/fabric/api/client/rendering/v1/FabricSkyPropertyBuilder.java

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.rendering.v1;
+
+import java.util.Objects;
+
+import net.minecraft.client.render.SkyProperties;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.Vec3d;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+/**
+ * Builder used to create {@link net.minecraft.client.render.SkyProperties} instances which
+ * govern various things about how a dimension renders including where thick fog should be used, what type of should be rendered, et ect.
```

Comment:
> what type of should be rendered

1\) Type of what? I guess this is missing a word. 2) Also:

```suggestion
 * govern various things about how a dimension renders including where thick fog should be used, what type of should be rendered, etc.
```